utils/get-git-status.js

```diff
@@ -0,0 +1,16 @@
+const execute = require('./execute')
+const assert = require('assert')
+
+async function getGitStatus () {
+  const lines = (await execute('git status')).stdout.split(/\r?\n/)
+
+  const match = lines[0].match(/^On branch (\S+)$/)
+  assert.ok(match, 'Unable to determine current branch')
+  const currentBranch = match[1]
+
+  const cleanWorkingTree = lines.find(line => line.startsWith('nothing to commit,'))
```

Comment:
The result should be converted to boolean, e.g.

```js
const cleanWorkingTree = !!lines.find(line => line.startsWith('nothing to commit,'))
```

Reply:
4e8d5482e9f743542e59812f9089db144eb83fb4